tests/test_exceptions.py

```diff
@@ -17,3 +18,15 @@ def test_serialize_to_json(self):
         deserialized = json.loads(mlflow_exception.serialize_as_json())
         assert deserialized['message'] == 'test'
         assert deserialized['error_code'] == 'INTERNAL_ERROR'
+
+    def test_get_http_status_code(self):
+        assert MlflowException('test', error_code=INVALID_STATE).get_http_status_code() \
```

Comment:
Should we test one that hits the default error code case?

Reply:
Added

Reply:
Ah sorry, I meant to test the case where the error_code doesn't match anything in your map.

Reply:
Got it - will add that. Frankly the case where no error code is declared is a good test case too, so I'll keep it in.